stripe/src/main/java/com/stripe/android/view/LockableHorizontalScrollView.java

```diff
@@ -50,4 +68,41 @@ public boolean onInterceptTouchEvent(MotionEvent event) {
         // Reject all intercept events if mScrollable is false.
         return mScrollable && super.onInterceptTouchEvent(event);
     }
+
+    void setScrollChangedListener(LockableScrollChangedListener listener) {
+        mLockableScrollChangedListener = listener;
+    }
+
+    /**
+     * Wrapping the {@link HorizontalScrollView#smoothScrollBy(int, int)} function to increase
+     * testability.
+     *
+     * @param dx the number of pixels to scroll by on the X axis
+     * @param dy the number of pixels to scroll by on the Y axis
+     */
+    void wrappedSmoothScrollBy(int dx, int dy) {
+        if (!mScrollable) {
+            return;
+        }
+
+        smoothScrollBy(dx, dy);
```

Comment:
Sadly, `HorizontalScrollView#smoothScrollBy` is `final`, so I couldn't just override it.